tools/js-tools/eslintrc/tests.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Why did you add this file but then only use it in one place, leaving the other 16 references to `plugin:jest/recommended` alone?

Reply:
Thank you for the catch. I have removed those 16 files and used `overrides` in base eslint config to set `jest/recommended` for the test files.